modules/nw-kernel-bonding.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * networking/advanced_networking/network-bonding-considerations.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-kernel-bonding_{context}"]
+= Kernel bonding
+
+[role="_abstract"]
+Kernel bonding is a built-in Linux kernel function where link aggregation can exist among many Ethernet interfaces to create a single logical physical interface. Kernel bonding is the default mode if no bond interfaces depend on OVS bonds. This bonding type does not give the same level of customization as supported OVS bonding.
+
+For `kernel-bonding` mode, the bond interfaces exist outside, which means they are not in the data path, of the bridge interface. Network traffic in this mode is not sent or received on the bond interface port but instead requires additional bridging capabilities for MAC address assignment at the kernel level. 
+
+If you enabled `kernel-bonding` mode on network controller interfaces (NICs) for your nodes, you must specify a Media Access Control (MAC) address failover. This configuration prevents node communication issues with the bond interfaces, such as `eno1f0` and `eno2f0`.
+
+Red Hat supports only the following value for the `fail_over_mac` parameter:
+
+* `0`: Specifies the `none` value and this is the default value that disables MAC address failover so that all interfaces receive the same MAC address as the bond interface.
+
+With the exception of {ibm-z-title} that supports value `1` for the `fail_over_mac` parameter, Red Hat does not support the following values for the `fail_over_mac` parameter on supported platforms:
```

Comment:
Sorry for being difficult, but it's better to not mention ibm-z here and keep the original "not-supported" language.     We can leave the `fail_over_mac=1` language to it's original location in the IBM-Z section.